tests/trace/test_client_feedback.py

```diff
@@ -228,7 +228,7 @@ def test_feedback_payload(client):
 def test_feedback_create_too_large(client):
     project_id = client._project_id()
 
-    value = "a" * 10000
+    value = "a" * 1024 * 1024
```

Comment:
I am surprised this passes since the size check is for a value > than this (not >=) but InvalidRequest still gets raised?